packages/contracts/contracts/test/PermissionManagerTest.sol

```diff
@@ -4,47 +4,43 @@ pragma solidity 0.8.10;
 
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
 
-import "../core/acl/ACL.sol";
+import "../core/permission/PermissionManager.sol";
 
-contract ACLTest is ACL {
+contract PermissionManagerTest is PermissionManager {
     function init(address _who) public initializer {
-        super.__ACL_init(_who);
+        super.__PermissionManager_init(_who);
     }
 
     function getAuthPermission(
         address _where,
         address _who,
-        bytes32 _role
+        bytes32 _permissionID
     ) public view returns (address) {
-        return authPermissions[permissionHash(_where, _who, _role)];
-    }
-
-    function getFreezePermission(address _where, bytes32 _role) public view returns (bool) {
-        return freezePermissions[freezeHash(_where, _role)];
+        return permissionsHashed[permissionHash(_where, _who, _permissionID)];
     }
 
     function getPermissionHash(
         address _where,
         address _who,
-        bytes32 _role
+        bytes32 _permissionID
     ) public pure returns (bytes32) {
-        return permissionHash(_where, _who, _role);
+        return permissionHash(_where, _who, _permissionID);
     }
 
-    function getFreezeHash(address _where, bytes32 _role) public pure returns (bytes32) {
-        return freezeHash(_where, _role);
+    function getImmutableHash(address _where, bytes32 _permissionID) public pure returns (bytes32) {
+        return immutablePermissionHash(_where, _permissionID);
     }
 
     function getAnyAddr() public pure returns (address) {
         return ANY_ADDR;
     }
 
-    function checkRole(
+    function checkPermission(
```

Comment:
Same as said earlier. `hasPermission` or `canPerform`

Reply:
Fixed already as part of https://github.com/aragon/core/pull/62#discussion_r928888184 and therefore, this  conversation can be closed.